playbooks/libvirt/openshift-cluster/tasks/configure_libvirt_storage_pool.yml

```diff
@@ -5,12 +5,15 @@
     state: directory
 
 - acl:
-    default: yes
+    default: "{{ item }}"
     entity: kvm
     etype: group
     name: "{{ libvirt_storage_pool_path }}"
     permissions: rwx
     state: present
+  with_items:
+    - no
+    - yes
```

Comment:
I had to actually look up the ACL module to understand what was going on here... Do you mind adding a comment to this task to describe what it is doing? Something like the following would clear up any confusion, I think:
```
# We need to set permissions on the directory and any items created under the directory, so we need to call the acl module with and without default set.
```